specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/PrivateEndpointConnectionsGet.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "privateEndpointConnectionName": "privateendpointconnection1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "privateEndpoint": {
+            "id": "/subscriptions/subid2/resourceGroups/rg2/providers/Microsoft.Network/privateEndpoints/pe2"
+          },
+          "privateLinkServiceConnectionState": {
```

Comment:
missing description and actiosRequired
[NRP Spec section 7.4:](https://microsoft.sharepoint.com/:w:/r/teams/WAG/AzureNetworking/_layouts/15/Doc.aspx?sourcedoc=%7B64F10150-563A-4715-9476-FFA24F9BD42E%7D&file=%5BNRP%5D%20Private%20Endpoint%20Design%203.docx&action=default&mobileredirect=true&cid=7075b01e-18b3-42f7-950a-e1a09ede0e17)

Reply:
these are AKS API instead of NRP API. They won't be invoked by NRP

Reply:
I know is AKS API, still those properties would be missing like all other Privately Linkable Resources